.github/workflows/package-manager-ci.yml

```diff
@@ -0,0 +1,97 @@
+name: package-manager-ci
+on:
+  push:
+    paths-ignore:
+        - 'docs/**'
+        - '*.md'
+  pull_request:
+    paths-ignore:
+        - 'docs/**'
+        - '*.md'
+jobs:
+  pnpm:
+    name: pnpm package manager on ${{ matrix.node-version }} ${{ matrix.os }}
+    runs-on: ${{ matrix.os }}
+    strategy:
+      fail-fast: false
+      matrix:
+        os: [macOS-latest, windows-latest]
+        node-version: [12, 14, 16]
+    steps:
+      - uses: actions/checkout@v2.3.4
+      - name: Use Node.js ${{ matrix.node-version }}
+        uses: actions/setup-node@v2.4.0
+        with:
+          node-version: ${{ matrix.node-version }}
+      - name: Use pnpm
+        uses: pnpm/action-setup@v2.0.1
+        with:
+          version: ^6.0.0
+      - name: Install dependancies
+        run: pnpm install
+      - name: Tests
+        run: pnpm run test-ci
+
+  yarn:
+    name: yarn package manager on ${{ matrix.node-version }} ${{ matrix.os }}
+    runs-on: ${{ matrix.os }}
+    strategy:
+      fail-fast: false
+      matrix:
+        os: [macOS-latest]
+        node-version: [12, 14, 16]
+    steps:
+      - uses: actions/checkout@v2.3.4
+      - name: Use Node.js ${{ matrix.node-version }}
+        uses: actions/setup-node@v2.4.0
+        with:
+          node-version: ${{ matrix.node-version }}
+      - name: Use yarn
+        run: |
+          npm install -g yarn
+          yarn set version berry && yarn set version 2.4.2
+          echo "nodeLinker: node-modules" >> .yarnrc.yml
+          # see https://github.com/yarnpkg/berry/issues/2935#issuecomment-911299992
+          yarn add --dev typescript@~4.3.2
```

Comment:
```suggestion
```
We've released the fix for this issue

Reply:
Thanks

To avoid this issue for future typescripts minor releases, where could we read the last working version integrated?

Reply:
We try to update the patch as soon as they make a new release so we don't have it documented anywhere. Also TypeScript doesn't follow semver so their `minor` is actually a `major` release

Reply:
> TypeScript doesn't follow semver so their minor is actually a major release

For this reason, the pino's maintainer may evaluate to pin the dep to `~4.4.0` so it will be dependabot to update it and trigger the new error for new minor changes